azurerm/internal/services/synapse/synapse_role_assignment_resource.go

```diff
@@ -55,9 +72,16 @@ func resourceSynapseRoleAssignment() *pluginsdk.Resource {
 				Required: true,
 				ForceNew: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					"Workspace Admin",
-					"Apache Spark Admin",
-					"Sql Admin",
```

Comment:
i think we'll still need to keep these around until 3.0 so the user doesn't have to change their config (prevent breaking change(

Reply:
and could we add a comment to be removed